### ISSUE TITLE
Style/EmptyLinesAroundAccessModifier autocorrection splits comment after access modifier

### ISSUE THREAD

**Issue by vergenzt** [bbatsov/rubocop#4268]
When an access modifier has a comment on the same line and is not followed by an empty line, the autocorrection for `Style/EmptyLinesAroundAccessModifier` inserts the newline *before* the comment. I would expect it to insert the newline after the comment.

(I discovered this while working on #4127 because this bug broke the `# rubocop:disable Lint/UselessAccessModifier` my change added to a line. I'm working on a fix.)

## Example
```rb
class Example
  something

  private # let's make the rest private
  def example; end
end
```

### Expected correction
```rb
class Example
  something

  private # let's make the rest private

  def example; end
end
```

### Actual correction
(one iteration)

```rb
class Example
  something

  private
 # let's make the rest private
  def example; end
end
```

## Steps to reproduce

From an updated `rubocop` clone:
1. `git checkout fd25562`
2. `curl https://gist.githubusercontent.com/vergenzt/ac933335208576084a9e464e42f12ad6/raw/f15b4b6cabbf603032e9cc111be9ba2e7f4a0c1f/spec.diff | git apply`
3. `rspec spec/rubocop/cop/style/empty_lines_around_access_modifier_spec.rb`

([Gist link](https://gist.github.com/vergenzt/ac933335208576084a9e464e42f12ad6))

## RuboCop version

```
$ rubocop -V
0.48.1 (using Parser 2.4.0.0, running on ruby 2.2.1 x86_64-darwin14)
```
